src/app/components/copy-to-clipboard/index.html

```diff
@@ -0,0 +1,40 @@
+<sky-demo-page
+  pageTitle="Copy to clipboard"
+  skyuxModule="SkyClipboardModule"
+  packageName="lib-clipboard"
+>
+
+  <sky-demo-page-summary>
+    <p>
+      The copy to clipboard component creates a button on your page that enables users to copy the content to their computer's clipboard.
+    </p>
+  </sky-demo-page-summary>
+
```

Comment:
When I serve locally, I see an Installation section with a code block that contains `npm install lib-clipboard --save --save-exact`. But when I look in the source file, I don't see that. Do you know what's up with that? We should have the install info in the note for now and not a separate section, but even if we decide on a separate section, this info seems incomplete.

Reply:
I updated the note to include `--save-exact` in the `npm install` command and also updated the `packageName` property so that the `npm install` command in the Installation section points to the correct path. I'll push that up to the PR shortly, but I'm going ahead and resolving this comment.